src/loader/bgw_launcher.c

```diff
@@ -502,17 +501,20 @@ launcher_pre_shmem_cleanup(int code, Datum arg)
  */
 
 /*
- * This should be idempotent. If we find the background worker and it's
- * not stopped, do nothing. If we have a worker that stopped
- * but hasn't been cleaned up yet, simply restart the worker.
+ * This should be idempotent. If we find the background worker and it's not
+ * stopped, do nothing. In order to make this action truly idempotent, we do not
```

Comment:
I don't understand "we do not change the vxid that a job is waiting on".

In my understanding, the only processes waiting on a`vxid` would be the schedulers at startup. Is a scheduler the same as a "job"? What does it mean for the `vxid` to change? I thought we just (optionally) set the `vxid` on start and then it is reset once started, so when does it change? Or, does "change" here just mean "set"?